src/rdkafka_partition.c

```diff
@@ -4421,6 +4421,91 @@ int rd_kafka_topic_partition_list_regex_cnt(
 }
 
 
+/**
+ * @brief Match function that returns true if topic is non regex.
+ */
+static RD_UNUSED int rd_kafka_topic_partition_not_regex(const void *elem,
```

Comment:
`RD_UNUSED` is usually together with `RD_INLINE` for functions in headers that can be included and used in other files
```suggestion
static int rd_kafka_topic_partition_not_regex(const void *elem,
```